src/Audio/components/VerseGrid/verse.js

```diff
@@ -0,0 +1,82 @@
+export default [
```

Comment:
What is this about?

Reply:
this is a sample verse file for the audio component used for stand-alone time. Instead of this component translation panel comes when we integrate with AG...

Reply:
@joelthe1 Anyways I've removed two components that I'm not going to use in the integrated AG